### DISCUSSION TITLE
Initial Setup not working 🤷🏻‍♂️

### DISCUSSION THREAD

**Discussion by rfmdnsrvc** [teslamate-org/teslamate#4232] [Getting started]
Hey everyone, I hope someone is able to help me! ✌🏼

I try to setup TeslaMate in Home Assistant. I installed Postgres, Grafana and TeslaMate via HA.

My Postgres and TelsaMate sample config is attached.

The log shows the following:
[13:56:59] INFO: Creating database 'teslamate' on db21ed7f-postgres
psql: error: connection to server at "*-postgres" (*), port 5432 failed: FATAL:  password authentication failed for user "postgres"
psql: error: connection to server at "*-postgres" (*), port 5432 failed: FATAL:  password authentication failed for user "postgres"
[13:56:59] ERROR: Failed to create database teslamate
[14:56:59] ERROR: TeslaMate encountered an error, halting add-on

So everywhere the same credentials are entered... what am I doing wrong? 🫣

![teslamate-config](https://github.com/user-attachments/assets/2c31fb25-bd70-487d-a3c6-29ad311a56ab)
![postgres-config](https://github.com/user-attachments/assets/83b3036f-dedd-469b-bf92-ff74d4012f74)

**Comment by KaVo194** [comment]
I have the same problem. Did you solve it?
